Tests/ApolloCodegenTests/ApolloCodegenTests.swift

```diff
@@ -1096,15 +1864,56 @@ class ApolloCodegenTests: XCTestCase {
   func test_validation_givenTestMockConfiguration_asSwiftPackage_withSchemaTypesModule_asSwiftPackage_shouldNotThrow() throws {
     // given
     let configContext = ApolloCodegen.ConfigurationContext(config: .mock(
-      input: .init(schemaPath: "path"),
-      output: .mock(
-        moduleType: .swiftPackageManager,
-        testMocks: .swiftPackage(targetName: nil)
-      )
+      input: .init(schemaPath: "path.graphqls")
```

Comment:
Should this initializer test include `operationSearchPaths` too so we know it's valid with an extension? The new validation tests only test for throwing on `operationSearchPaths` without an extension.

Reply:
I'm saying we don't have a happy path test with the inclusion of `operationSearchPaths`, only throwing tests. It works now because the default value of `operationSearchPaths` in this convenience initializer is `["**/*.graphql"]`. I was suggesting this line become `input: .init(schemaPath: "path.graphqls", operationSearchPaths: ["**/*.graphql"])` to become more explicit about it validating when both `schemaPath` and `operationSearchPaths` include an extension.

Reply:
The default value proves the test though and if that ever changed this test would likely break. I'm OK with that.